js/foam/apps/calc/CalcView.js

```diff
@@ -160,8 +161,10 @@ CLASS({
         const history = Array(historyNodeList.length).fill(0).map((_,i) => historyNodeList[i])
 
         // remove the equals speech label from first history elem
-        if (history[0])
-          history[0].setAttribute('aria-label', history[0].getAttribute('aria-label').replace(/^=/,''))
+        const equals = window.chrome.i18n ? window.chrome.i18n.getMessage('Calc_ActionSpeechLabel_equals') : 'equals';
+
+        if (history[0] && history[0].getAttribute('aria-label'))
```

Comment:
Could you help me understand what's happening here? Potentially a comment in code is appropriate, just trying to figure out what you're doing.

Reply:
I chucked in a comment :)

```
        The way the history is set up means that every element in the history
        has to be rendered the same way, as such every other number element has
        the aria label "equals <num>". This means as you are tabbing through
        something such as
         A
         + B
        --------
         C
        The screen reader helpfully announces "A + B equals C" as "C" has
        the "equals C" aria label. Sadly we don't want this label on A, only
        on C and any subsequent numbers. This bit of JS removes that label
        from the first element in the history.
```